### DISCUSSION TITLE
How to add more than 7 different languages?

### DISCUSSION THREAD

**Discussion by jaads** [directus/directus#15666] [Archive]
On the Directus website it says "Multi-lingual support for 50+ languages ensures consistency across geographies".

But when I want to add translation for field, I can only choose from 7 different languages. 

Is there a way I can extend the list of language-codes? 

![image](https://user-images.githubusercontent.com/34009329/191282450-33ba822f-2e89-4376-9479-6aa105e7d02c.png)

**Comment by br41nslug** [comment]
Adding the translations interface will also add a `languages` collection where these are housed. You add/remove/edit them there as you please 😄 
You can also have multiple collections for languages and choose which to use when creating the interface.

**Reply by jaads** [reply]
Oh wow so obvious. I wasn't thinking clear :smile: Thanks for the quick response!

**Reply by rijkvanzanten** [reply]
ngl for a second I thought @licitdev updated his avatar on GH 😂

**Reply by licitdev** [reply]
Oh yes! "Same Same, But Different" 😆

**Reply by jaads** [reply]
quite similar, that's for sure :smile:
